ocean_provider/utils/util.py

```diff
@@ -198,13 +198,17 @@ def get_compute_result_endpoint():
     return get_config().operator_service_url + "/api/v1/operator/getResult"
 
 
-def get_compute_address():
+def get_compute_info():
     try:
+        limits = dict()
         compute_info = requests.get(get_config().operator_service_url).json()
-        return compute_info.get("address", None)
+        limits["algoTimeLimit"] = compute_info.get("algoTimeLimit", None)
+        limits["storageExpiry"] = compute_info.get("storageExpiry", None)
+        compute_address = compute_info.get("address", None)
+        return compute_address, limits
```

Comment:
I don't like the fact that we are returning a tuple, build from a dictionary. We can return the dictionary with "algoTimeLimit", "storageExpiry" and "address" and it would make more sense.

Reply:
we need a tuple, because that function is used in multiple places, and there is no point in returning a dict and then pop out values out of it based on needs